src/onegov/swissvotes/models/policy_area.py

```diff
@@ -112,19 +112,16 @@ def label_path(self) -> list[str]:
         for part in self.descriptor_path:
             lookup = lookup.get(part)
             if not lookup:
-                # FIXME: Why do we have different fallbacks?
                 result.append(str(self.descriptor_decimal))
                 break
-            # FIXME: Why do we have different fallbacks?
             result.append(lookup.label or str(self.descriptor))
         return result
 
     def html(self, request: 'SwissvotesRequest') -> str:
-        # FIXME: Use Markup
         title = ' &gt; '.join(
             request.translate(part) for part in self.label_path
         )
-        return f'<span>{title}</span>'
+        return Markup(f'<span>{title}</span>')  # noqa: MS001
```

Comment:
```suggestion
        return Markup('<span>{}</span>').format(title)
```
Auch hier, du kannst oben entweder von &gt; zu `>` wechseln, oder `Markup(...).join` machen damit `>` korrekt dargestellt wird, sollte beides funktionieren.